docs/tutorial/import_export.rst

```diff
@@ -0,0 +1,1021 @@
+
+.. code-block:: yaml
+
+    - id: ACALD
+      name: acetaldehyde dehydrogenase (acetylating)
+      genes: b0351 or b1241
+      equation: '|acald_c[c]| + |coa_c[c]| + |nad_c[c]| <=> |accoa_c[c]| + |h_c[c]| +
+        |nadh_c[c]|'
+      subsystem: Pyruvate Metabolism
+    - id: ACALDt
+      name: acetaldehyde reversible transport
+      genes: s0001
+      equation: '|acald_e[e]| <=> |acald_c[c]|'
+      subsystem: Transport, Extracellular
+
+
+Each reaction entry is designated with the reaction ID first. Then the various
+properties of the reaction can be listed below it. The required properties for
+a reaction are ID and equation. Along with these required attributes others
+can be included as needed in a specific project. These can include but are not
+limited to EC numbers, subsystems, names, and genes associated with the
+reaction. For example, in a collaborative reconstruction you may want to
+include a field named ``authors`` to identify which authors have contributed to
+the curation of a particular reaction.
+
+Reaction equations can be formatted in multiple ways to allow for more
+flexibility during the modeling process. The reactions can be formatted in a
+string format based on the ModelSEED reaction format. In this representation
+individual compounds in the reaction are represented as compound IDs followed by
+the cellular compartment in brackets, bordered on both sides by single pipes.
+For example if a hydrogen compound, ``Hydr``, in a ``cytosol`` compartment
+was going to be in an equation it would be represented as follows:
+
+.. code-block:: shell
+
+    |Hydr[cytosol]|
+
+These individual compounds can be assigned stoichiometric coefficients by
+adding a number in parentheses before the compound. For example if a reaction
+contained two hydrogens it could appear as follows:
+
+.. code-block:: shell
+
+    (2) |Hydr[cytosol]|
+
+These individual components are separated by + signs in the reaction string. The
+separation of the reactants and products is through the use of an equal sign
+with greater than or less than signs designating directionality. These could
+include => or <= for reactions that can only progress in one direction or <=>
+for reactions that can progress in both directions. An example of a correctly
+formatted reaction could be as follows:
+
+.. code-block:: shell
+
+    '|ac_c[c]| + |atp_c[c]| <=> |actp_c[c]| + |adp_c[c]|'
+
+For longer reactions the YAML format
+provides a way to list each reaction component on a single line. For example a
+reaction could be represented as follows:
+
+.. code-block:: yaml
+
+    - id: ACKr
+      name: acetate kinase
+      equation:
+        compartment: c
+        reversible: yes
+        left:
+          - id: ac_c
+            value: 1
+          - id: atp_c
+            value: 1
+        right:
+          - id: actp_c
+            value: 1
+          - id: adp_c
+            value: 1
+      subsystem: Pyruvate Metabolism
+
```

Comment:
This also has `_c` suffixes on compound IDs. I'm not sure if we should remove them since compound merging is not yet on by default in psamm-import?